ee/codegen/src/__test__/nodes/generic-nodes/__snapshots__/generic-node.test.ts.snap

```diff
@@ -52,6 +61,11 @@ from ...nodes.my_custom_node import MyCustomNode
 @BaseRetryNodeDisplay.wrap(node_id=UUID("adornment-3"))
 class MyCustomNodeDisplay(BaseNodeDisplay[MyCustomNode]):
     node_input_ids_by_name = {}
+    port_displays = {
+        MyCustomNode.Ports.default_port: PortDisplayOverrides(
```

Comment:
[nit] we should update our `nodePortFactory` to make sure the DEFAULT port always uses the `default` attribute name